Leanplum-iOS-SDK-Example.xcodeproj/project.pbxproj

```diff
@@ -48,8 +49,11 @@
 		07DAAE241E78499B002A6878 /* ABTestViewController.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ABTestViewController.m; sourceTree = "<group>"; };
 		07DAAE261E7849A9002A6878 /* PreviewViewController.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = PreviewViewController.h; sourceTree = "<group>"; };
 		07DAAE271E7849A9002A6878 /* PreviewViewController.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = PreviewViewController.m; sourceTree = "<group>"; };
+		682C0D5520A3A74D004649A4 /* Leanplum-iOS-SDK-Example.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.plist.entitlements; path = "Leanplum-iOS-SDK-Example.entitlements"; sourceTree = "<group>"; };
 		68C9A5AF1F143832000DFA1B /* TestingViewController.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = TestingViewController.h; sourceTree = "<group>"; };
 		68C9A5B01F143972000DFA1B /* TestingViewController.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = TestingViewController.m; sourceTree = "<group>"; };
+		68DC839820A20EC500708F92 /* LPNotification.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = LPNotification.m; sourceTree = "<group>"; };
+		68DC839A20A26A1600708F92 /* LPNotification.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = LPNotification.h; sourceTree = "<group>"; };
```

Comment:
Let's rename the file LeanplumExampleTestNotification... reserve the LP namespace for the SDK

Reply:
Yes will do, based on our discussions from the other day.